content/en/docs/concepts/storage/volumes.md

```diff
@@ -244,9 +244,8 @@ The `emptyDir.medium` field controls where `emptyDir` volumes are stored. By
 default `emptyDir` volumes are stored on whatever medium that backs the node
 such as disk, SSD, or network storage, depending on your environment. If you set
 the `emptyDir.medium` field to `"Memory"`, Kubernetes mounts a tmpfs (RAM-backed
-filesystem) for you instead.  While tmpfs is very fast, be aware that unlike
-disks, tmpfs is cleared on node reboot and any files you write count against
-your container's memory limit.
+filesystem) for you instead.  While tmpfs is very fast be aware that, unlike
+disks, files you write count against your container's memory limit.
```

Comment:
```suggestion
disks, files you write count against the memory limit of the container that wrote them.
```
AIUI it's even more complicated than this. If a container exists, the Pod's cgroup ends up owning the memory used to back the tmpfs. Another PR could write that up.